### ISSUE TITLE
No fid_kid.py

### ISSUE THREAD

**Issue by naoto0804** [alpc91/NICE-GAN-pytorch#5]
I want to compute FID/KID, but it seems there is no `fid_kid.py`. Is it possible to share?

**Comment by alpc91** [comment]
Please cd ./metirc

**Comment by naoto0804** [comment]
Thanks! Oh, I've missed such an easy thing;

**Comment by naoto0804** [comment]
What's the FID/KID score of the provided cat2dog model?

When I compute the score of it, 
```
$ python metric/fid_kid.py dataset/cat2dog/testB results/cat2dog/fakeB --mmd-var
```
FID:  31.21598867148849
mean MMD^2 estimate: 0.31708042463877906

```
$ python metric/fid_kid.py dataset/cat2dog/testA results/cat2dog/fakeA --mmd-var
```
FID:  43.559898830804855
mean MMD^2 estimate: 1.1181057541950077

This is too good compared to Table. 1 in the paper.
Is this just from different seed, or am I missing something?

**Comment by naoto0804** [comment]
@alpc91 sorry to bother you, but I would really appreciate it if you reply.

**Comment by alpc91** [comment]
The provided pretrianed model is about trained on more than 200K iters. And different seeds will also have some impact on the results.

> @alpc91 sorry to bother you, but I would really appreciate it if you reply.
